apps/judicial-system/backend/src/app/modules/notification/test/internalNotificationController/sendCaseFilesUpdatedNotifications.spec.ts

```
@@ -0,0 +1,152 @@
+                spokespersonName,
+                spokespersonEmail,
+              },
+            ],
+          } as Case,
+          {
+            user,
+            type: NotificationType.CASE_FILES_UPDATED,
+          },
+        )
+        .then((result) => (then.result = result))
+        .catch((error) => (then.error = error))
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)